homeassistant/components/light/hue.py

```diff
@@ -161,12 +171,26 @@ def name(self):
     @property
     def brightness(self):
         """ Brightness of this light between 0..255. """
-        return self.info['state']['bri']
+        if 'bri' in self.capabilities:
+            return self.info['state']['bri']
```

Comment:
I'm not too familiar with the returned state from Hue, but would the returned state contain a `bri` key if the light does not support brightness ? If that is the case, this method could just be this: (`get()` returns None if key not found and no default argument supplied):
```python
def brightness(self):
    return self.info['state'].get('bri')
```